functions.js

```diff
@@ -0,0 +1,129 @@
+// ## Question One
+// Write a function named double that takes in a number and returns that number times two
+
+// const double=num=>{
+//     return num*2;
+// }
+// console.log(double(3));
+
+// ## Question Two
+// Write a function named smallest that takes in two numbers and returns the smaller number
+
+// const smallest=(num1,num2)=>{
+//     if(num1<num2){
+//         return num1;
+//     }else if(num2===num1){
+//         console.log("the numbers are equal");
+//     }else if(num1>num2){
+//         return num2;
+//     }
+// }
+// console.log(smallest(7,3));
+
+// ## Question Three
+// Write a function named smallestValueInArr 
+// that takes in an array of numbers and returns the smallest number
+
+// const smallestValueInArr = arr =>{
+//      return Math.min(...arr)
+// }
+// console.log(smallestValueInArr([-4,-59,-348,-34,-4]));
+
+// ## Question Four
+// Write a function named occurrences that counts how many characters in a string match a specific character.
+
+// const occurrences = (str, char) =>{
+//     let count=0;
+//     for(let i=0; i<str.length; i++){
+//         if(str[i]===char){
+//             count++
+//         }
+//     }
+//     return count;
+// }
+// console.log(occurrences("hello", "l"));
+
+// ## Question Five
+// Write a function named average that returns the average of an array of numbers
+
+// const average = arr1 =>{
+//     let total = 0;
+//     for(let i=0; i<arr1.length; i++){
+//         total+=arr1[i];
+//     }
+//     let aver = total/arr1.length;
+//     return aver;
+// }
+// console.log(average([-10,5,-15,20]));
+
+// ## Question Six
+// Write a function named frequencyMap that takes a string as input and returns an object 
+// that maps each character its number of occurrences
+
+// const counter= str =>{
+// let count=0;
+//     for(let i=0; i<str.length; i++){
+//         if(str[i]===char){
+//             count++
+//         }
+// }        
+
+// const frequencyMap = str =>{
+//     str.map(counter())
+// }
```

Comment:
You need to return an object with counts. {h: 1, e: 1, l: 2, o: 1}  is what it would look like for hello. Yours is calling the array map method on a string. Please revisit.